DOCS-6851-added new content (#18)
mulesoft/docs-service-mesh: modules/ROOT/pages/provision-adapter-configure-service-mesh-CRD.adoc

= Provision the Adapter Using the CRD
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:keywords: service mesh, microservices, installation prerequisites

> Before you can start managing your services using Anypoint Service Mesh, you must prepare your namespace by provisioning a Anypoint Service Mesh adapter. Provisioning prepares the namespace for the Anypoint Service Mesh configuration. As a best practice, you must create one namespace per adapter.

Reviewer: ```suggestion
Before you can start managing your services using Anypoint Service Mesh, you must prepare your namespace by provisioning an Anypoint Service Mesh adapter. Provisioning prepares the namespace for the Anypoint Service Mesh configuration. As a best practice, you must create one namespace per adapter.
```

Author: Done! Thanks!

> The adapter behaves as the policy enforcement component, which fetches policies from Anypoint platform, checks whether the policies have been met, and then reports the analytics data back to Anypoint Platform.  

Reviewer: ```suggestion
The adapter acts as a policy enforcement component that fetches policies from Anypoint platform, checks whether the policies have been met, and then reports the analytics data back to Anypoint Platform.  
```

Author: Done! Thanks!
